src/components/VoiceDemo/VoiceDemo.js

```
@@ -0,0 +1,122 @@
+                    onStart: function () {
+                        gsap.set(this.targets(), { opacity: 0.4 });
+                    }
+                });
+            };
+
+            let pulseAnimations = [];
+
+            // Effect to handle speaker rotation
+            const rotateSpeakers = () => {
+                // Kill previous animations
+                pulseAnimations.forEach(anim => anim.kill());
```

Biome (2.3.14)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)